react-native-aztec/android/src/test/java/org/wordpress/mobile/ReactNativeAztec/IllegalSelectionIndexExceptionTest.java

```diff
@@ -0,0 +1,18 @@
+package org.wordpress.mobile.ReactNativeAztec;
+
+import org.junit.Test;
+
+import java.util.Collections;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class IllegalSelectionIndexExceptionTest {
+
+    @Test
+    public void getHtmlTag_returns_only_tag() {
```

Comment:
Sub-nit (yes, there can one nested nits 😸), WDYT about `s/getHtmlTag/parseTags`?

Reply:
Yep, I missed that rename. Good catch! It's been updated.